agent/lib/kontena/observable.rb

```diff
@@ -0,0 +1,72 @@
+module Kontena
+  # An Actor that has some value
+  # The value does not yet exist when initialized, it is nil
+  # Once the value is first updated, then other Actors will be able to observe it
+  # When the value later updated, other Actors will also observe those changes
+  module Observable
+    include Kontena::Logging
+
+    def value
```

Comment:
Probably too common name to have in a included module? Maybe `observable_value`?